src/Controls/tests/DeviceTests/Elements/Image/ImageTests.cs

```diff
@@ -46,5 +47,28 @@ public async Task ImageWithUndefinedSizeAndWithBackgroundSetRenders()
 				await handler.ToPlatform().AssertContainsColor(Colors.Red, MauiContext);
 			});
 		}
+
+		[Fact]
+		public async Task ImageSetFromStreamRenders()
+		{
+			SetupBuilder();
+			var layout = new VerticalStackLayout();
+
+			var stream = GetType().Assembly.GetManifestResourceStream("Microsoft.Maui.DeviceTests.Resources.Images.red-embedded.png");
```

Comment:
Then you can use `GetManifestResourceStream("red-embedded.png")`.

You might want to put `using`, too:

```suggestion
			using var stream = GetType().Assembly.GetManifestResourceStream("red-embedded.png");
```